x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_sidepanel/index.tsx

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
The `conversation_sidepanel` dir needs unit tests, let's make sure these are added after FF